account_move_line_tax_editable/views/account_move.xml

```diff
@@ -14,12 +14,8 @@
                 <field name="is_tax_editable" invisible="1" />
                 <field
                     name="tax_line_id"
-                    attrs="{'readonly': [('is_tax_editable', '=', False)], 'invisible': [('parent.type', '!=', 'entry')]}"
-                />
-                <field
-                    name="tax_ids"
-                    attrs="{'readonly': [('is_tax_editable', '=', False)], 'invisible': [('parent.type', '!=', 'entry')]}"
-                    widget="many2many_tags"
+                    attrs="{'readonly': [('is_tax_editable', '=', False)], 'invisible': [('parent.type', '=', 'entry')]}"
```

Comment:
Why hiding it in any case? I would love to have it (both actually) for manual/imported entries.

Reply:
```suggestion
                    attrs="{'readonly': [('is_tax_editable', '=', False)]"
```

I also don't understand why you remote tax_ids field

Reply:
It seems it's already in core, Miquel.

Reply:
What I would do is to change its `attr` for not hiding it if manual entry.

Reply:
tax_ids has no invisible `attr` set on that view

Reply:
OK, if no `attrs` in core, then this one should be the same and all OK then.

Reply:
@pedrobaeza @MiquelRForgeFlow comments attended